From 80348eab3a9475c0e27abe6fbdc10c084f1d5a3a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Mon, 14 Oct 2024 23:05:57 +0000
Subject: [PATCH] build:  Pass cargo_options to cargo test

This is needed when running the tests out-of-tree.
---
 src/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/meson.build b/src/meson.build
index 7734654a..c5a8cb6e 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -60,7 +60,7 @@ cargo_home = cargo_target_dir / 'cargo-home'
 test (
   'Cargo tests',
   cargo,
-  args: ['test', '--bin', meson.project_name()],
+  args: ['test', '--bin', meson.project_name(), cargo_options],
   timeout: 600, # cargo might take a bit of time sometimes
   env: ['CARGO_TARGET_DIR=@0@'.format(cargo_target_dir), 'CARGO_HOME=@0@'.format(cargo_home)]
 )
-- 
GitLab

